_includes/markdown/Systems.md

```diff
@@ -252,3 +253,94 @@ Once tuning moves beyond these main items, performance improvements will be mino
 
 Most Linux servers have a few tmpfs mounts already. The `/dev/shm` mount is a tmpfs mount for efficiently storing temporary files for programs and we can set MySQL to use this with the tmpdir variable in the ```my.cnf``` file. tmpfs only uses space in RAM when files exist (it isn’t preallocated), but if the WordPress database is very busy, beware that MySQL could use as much RAM as is allocated for ```/dev/shm``` (which is normally 50% of total RAM). In most WordPress workloads, MySQL only uses a few megabytes in temporary tables on disk, but if ```/dev/shm``` usage were to grow, it could quickly cause an out of memory situation. Take this into account when tuning MySQL for memory usage. ```/dev/shm``` size should be monitored if used for the MySQL tmpdir.
+
+<h2 id="memcached-and-redis" class="anchor-heading">Memcached and Redis {% include Util/link_anchor anchor="memcached-and-redis" %} {% include Util/top %}</h2>
+
+Memcached and Redis are in memory data stores that are used for the WordPress object cache. Implementation of the object cache in WordPress code is covered extensively in the [PHP Performance](https://10up.github.io/Engineering-Best-Practices/php/#performance) section of the Best Practices and this section will focus on the hosting and setup of memcached and Redis.
+
+Memcached and Redis are used by WordPress as simple, in memory key-value stores. By being in memory and not having the possibility for complex queries, these tools provide blazing fast retrieval of data, usually in less than 1 millisecond. Common use cases of these data stores are:
+
+* Caching results of an external API call
+* Caching the results of a complex query
+* Storing full or partial pages 
+
+Items stored in these caching technologies are made up of 3 things: a key, a value, and an optional expiration time.  After the expiration time, items are evicted from the cache.  Items without an expiration time will persist in the cache until the cache is full, at which time the cache will evict the least recently used (LRU) item each time a new item needs to be stored. 
```

Comment:
Possibly a good idea to mention that while LRU is commonly the default there are other eviction policies

Reply:
Good call